packages/cli/test/integration.test.ts

```diff
@@ -1148,17 +1152,17 @@ test('Deploy `api-env` fixture and test `vercel env` command', async () => {
       }
     );
 
-    expect(exitCode).toBe(0);
+    expect(exitCode, formatOutput({ stdout, stderr })).toBe(0);
 
     const contents = fs.readFileSync(path.join(target, '.env'), 'utf8');
 
     const lines = new Set(contents.split('\n'));
 
-    expect(lines.has('VERCEL="1"')).toBe(true);
-    expect(lines.has('VERCEL_URL=""')).toBe(true);
-    expect(lines.has('VERCEL_ENV="development"')).toBe(true);
-    expect(lines.has('VERCEL_GIT_PROVIDER=""')).toBe(true);
-    expect(lines.has('VERCEL_GIT_REPO_SLUG=""')).toBe(true);
+    expect(lines).toContain('VERCEL="1"');
+    expect(lines).toContain('VERCEL_URL=""');
+    expect(lines).toContain('VERCEL_ENV="development"');
+    expect(lines).toContain('VERCEL_GIT_PROVIDER=""');
+    expect(lines).toContain('VERCEL_GIT_REPO_SLUG=""');
```

Comment:
Does toContain() work on Set? If it does thats great but I suspect it will fail.